Library/Formula/opentsdb.rb

```diff
@@ -0,0 +1,99 @@
+class Opentsdb < Formula
+  desc "Scalable, distributed Time Series Database."
+  homepage "http://opentsdb.net/"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.2.0/opentsdb-2.2.0.tar.gz"
+  sha256 "5689d4d83ee21f1ce5892d064d6738bfa9fdef99f106f45d5c38eefb9476dfb5"
+
+  option "without-lzo", "Don't use LZO Compression"
+
+  depends_on "hbase" if build.without? "lzo"
+  depends_on "hbase" => "with-lzo" if build.with? "lzo"
+  depends_on :java => "1.6+"
+  depends_on "gnuplot" => :optional
+
+  def confdir
+    etc/"opentsdb"
+  end
+
+  def cachedir
+    var/"cache/opentsdb"
+  end
+
+  def hbasedir
+    Formula["hbase"].opt_libexec.to_s
+  end
+
+  def hbase_plist
+    Formula["hbase"].plist_name
+  end
+
+  def install
+    inreplace "Makefile.in" do |s|
+      s.sub!(/(\$\(jar\): manifest \.javac-stamp) \$\(classes\)/, '\1')
+      s.sub!(/(echo " \$\(mkdir_p\) '\$\$dstdir'"; )/, '\1../')
+    end
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!